Add ZIP artifacts to the Kafka Connect Build docs (#5917)
strimzi/strimzi-kafka-operator: documentation/api/io.strimzi.api.kafka.model.connect.build.Build.adoc

<2> (Required) URL from which the archive is downloaded.
<3> (Optional) SHA-512 checksum to verify the artifact.

.Using ZIP artifacts

ZIP artifacts are used to download ZIP compressed archives.

> ZIP artifacts can be used in the same way as the TGZ artifacts described in the previous section.

Reviewer: ```suggestion
Use ZIP artifacts in the same way as the TGZ artifacts described in the previous section.
```

> The only difference is that instead of `type: tgz`, `type: zip` is used.

Reviewer: ```suggestion
The only difference is you specify `type: zip` instead of  `type: tgz`.
```
